docs/decision-maker-transaction.md

```diff
@@ -182,20 +182,16 @@ class TransactionHandler(Handler):
         :return: None
         """
         tx_msg_response = cast(TransactionMessage, message)
-        self.context.logger.info(tx_msg_response)
+        logger.info(tx_msg_response)
```

Comment:
@Totoual please revert to the `self.context.` approach which was introduced by Marco in a recent PR

Reply:
It results to an error that the `str has no info`

Reply:
Actually it is not possible to do this based on the way I am instantiating the TransactionHandler. 
`tx_handler = TransactionHandler(skill_context="skill_context", name="fake_skill")`
The way around would be to create a skill_context and pass it to the handler.

Reply:
ah ok, no fine, totally ok then